### PR TITLE
fix published key decrypt

### PR DESCRIPTION
**Pull request by sjpadgett**
fix error percolation to interface
Will leave open for a minute to maybe write a readme

### CODE REVIEW

**Comment by sjpadgett** [comment]
@bradymiller I did a boo boo updating my git. Please check this commit for me to ensure I recovered okay. Go ahead and bring in if you would after looking over. thanks sir...

**Comment by bradymiller** [comment]
hi @sjpadgett ,
Looks good and bringing in now. One of the nice things of the 'Squash and merge' for the github PR's is that it squashes branches that go "bad" into 1 commit :)
-brady

File: portal/portal_payment.php [hunk 1]
```diff
@@ -554,10 +554,10 @@ function notifyPatient() {
         var chargeMsg = <?php $amsg = xl('Payment was successfully authorized and your card is charged.') . "\n" .
                 xl("You will be notified when your payment is applied for this invoice.") . "\n" .
                 xl('Until then you will continue to see payment details here.') . "\n" . xl('Thank You.');
-            echo json_encode($amsg); // backward compatable 5.0.1
+            echo json_encode($amsg);
             ?>;
-        var publicKey = <?php echo json_encode($GLOBALS['gateway_public_key']); ?>;
-        var apiKey = <?php echo json_encode($GLOBALS['gateway_api_key']); ?>;
+        var publicKey = <?php echo json_encode(decryptStandard($GLOBALS['gateway_public_key'])) ?>;
```
Comment (bradymiller) [inline portal/portal_payment.php]: gonna add a very minor commit to add the above semicolon back

Comment (bradymiller) [reply portal/portal_payment.php]: yes, I missed something :)

**Pull request merged**